### ISSUE TITLE
Stopped working, warning msgs on re-install

### ISSUE THREAD

**Issue by ken-snyder** [drdelaney/OctoPrint-RepetierInformer#9]
Hi,

I've been using Repetier Informer app with OctoPrint for awhile.  I've been trying to keep my OctoPrint and OctoPi up to date, but at some point it stopped working. I've rebuilt my OctoPrint/Pi versions back to a level I know worked.  I've uninstalled and re-installed the OctoPrint Repetier Informer plug-in and get the following 3 warning msgs:

/home/pi/oprint/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#snimissingwarning.
SNIMissingWarning

/home/pi/oprint/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
InsecurePlatformWarning

/home/pi/oprint/local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
InsecurePlatformWarning

Given these msgs, I surmised that I need python 2.7.9 or later to get SNI, etc.  The OctoPi v 0..4 is using python 2.7.3.  Was there an earlier version of Repetier Informer plug-in that worked with this version?  If so, where can I download it?

Thanks

**Comment by drdelaney** [comment]
A stock OctoPi 0.14 image looks to contain python 2.7.9, and an upgraded 0.14 from Jessie to Stretch also seems to contain the 2.7.9 version. Or are you meaning OctoPi 0.4?  I could only find information on 0.5, which is very old.
I have tested and used RepetierInformer on a stock 0.14 install, so there should be no issues. I have not tested previous releases. If you have a link to an image of a previous release, I would be willing to try to see if I can get this to work on it, it really does nothing more than handle a URL request to a HTTPS enabled API.

**Comment by ken-snyder** [comment]
I found the problem.  I was still on OctoPi 0.12 that runs python 2.7.3.  That fixed it.  
Thanks,Ken
    On Tuesday, April 17, 2018, 9:31:18 PM PDT, Dan Delaney <notifications@github.com> wrote:  
 
 
A stock OctoPi 0.14 image looks to contain python 2.7.9, and an upgraded 0.14 from Jessie to Stretch also seems to contain the 2.7.9 version. Or are you meaning OctoPi 0.4? I could only find information on 0.5, which is very old.
I have tested and used RepetierInformer on a stock 0.14 install, so there should be no issues. I have not tested previous releases. If you have a link to an image of a previous release, I would be willing to try to see if I can get this to work on it, it really does nothing more than handle a URL request to a HTTPS enabled API.
